pymc3/distributions/transforms.py

```diff
@@ -128,19 +135,19 @@ def jacobian_det(self, x):
 class LogExpM1(ElemwiseTransform):
     name = "log_exp_m1"
 
-    def backward(self, x):
-        return aet.nnet.softplus(x)
+    def backward(self, rv_var, rv_value):
+        return aet.nnet.softplus(rv_value)
```

Comment:
Are the `rv_var` and `rv_value` mistaken for one another?
Before `x` was the tensor that the transformation was applied to. Reading the docstring:
```
        rv_var
            The random variable being transformed
        rv_value
            The parameters required for the transform.
```
I expected this line to read `aet.nnet.softplus(rv_var)`.

If I'm right this is switched around for multiple transforms above and below too. Otherwise please clarify :)

Reply:
The docstrings are wrong; they're result of a few revisions.  I'll fix that.